### PR TITLE
CLDC-812 rent type validation

### PR DESCRIPTION
**Pull request by dushan-madetech**
Screenshot 2022-03-16 at 10 47 36

### CODE REVIEW

**Comment by baarkerlounger** [comment]
Can you also add a spec that tests that new validation method gets called on CaseLog update - check the #update describe block.

**Review by smeny — APPROVED** [review]

**Review by smeny — CHANGES_REQUESTED** [review]
Just to be sure Dan's comment is visible :)

File: spec/models/case_log_spec.rb [hunk 1]
```diff
@@ -56,6 +56,10 @@
       expect(validator).to receive(:validate_startdate)
     end
 
+    it "validates intermediate rent product name" do
```
Comment (dushan-madetech) [inline spec/models/case_log_spec.rb:59]: @baarkerlounger Is this what you had in mind?

File: spec/models/case_log_spec.rb [hunk 2]
```diff
@@ -56,6 +56,10 @@
...
+      expect(validator).to receive(:validate_intermediate_rent_product_name)
+    end
+
```
Comment (dushan-madetech) [inline spec/models/case_log_spec.rb:62]: @baarkerlounger Is this what you had in mind?

**Review by baarkerlounger — APPROVED** [review]
Nice one

**Pull request merged**